appcenter-crashes/Crashes.js

```diff
@@ -122,8 +122,9 @@ const Helper = {
             return;
         }
         filteredReports.forEach((report) => {
-            const attachments = getErrorAttachmentsMethod(report);
-            AppCenterReactNativeCrashes.sendErrorAttachments(attachments, report.id);
+            getErrorAttachmentsMethod(report)
+            .then(attachments => AppCenterReactNativeCrashes.sendErrorAttachments(attachments, report.id))
+            .catch(error => AppCenterLog.error(LOG_TAG, 'Crashes.getErrorAttachments: ' + error));
```

Comment:
`.catch()` will catch any error thrown from the Promise chain. In other words, it can be error from `getErrorAttachmentsMethod()` or `sendErrorAttachments()`. You need to improve this error message.